config/environments/demo.rb

```diff
@@ -71,5 +71,6 @@
   # Set to :debug to see everything in the log.
   config.log_level = :info
 
-  config.logger = ActiveSupport::TaggedLogging.new(Le.new(ENV['LOGENTRIES_TOKEN'])) if ENV['LOGENTRIES_TOKEN'].present?
+  # Use a different logger for distributed setups.
+  config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
```

Comment:
can we throw this in `application.rb` since it's the same errwhere?

Reply:
Well, it's not the same in `test` and `development`. But I could do the inverse -- is there a `Logger.new` for writing to `logs/X.log`?

Reply:
except for `development` and `test`?

Reply:
oh, right. derp. nvm, let's roll with this.